src/Cms/HasFiles.php

```diff
@@ -90,6 +92,11 @@ public function file(string $filename = null, string $in = 'files')
 			return $this->$in()->first();
 		}
 
+		// find by global UUID
+		if (Str::contains($filename, 'file://') === true) {
```

Comment:
Shouldn't we use Str::startsWith here? Or maybe even have a global method that simplifies this.

Reply:
Yea, probably something like `Binding::isUuid(string|null $type = null)`

Reply:
Exactly!